docs/hub/security-jfrog.md

```diff
@@ -0,0 +1,27 @@
+# Third-party scanner: JFrog
+
+
+[JFrog](https://jfrog.com/)'s security scanner detects malicious behavior in machine learning models.
+
+![JFrog report for the danger.dat file contained in mcpotato/42-eicar-street](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/jfrog-report.png)
+*Example of a report for [danger.dat](https://huggingface.co/mcpotato/42-eicar-street/blob/main/danger.dat)*
+
+We partnered with JFrog to provide scanning in order to make the Hub safer. Model files are scanned by the JFrog scanner and reports you the scanning result.
```

Comment:
=> 

```
Model files are scanned by the JFrog scanner and we expose the scanning results on the Hub interface.
```

Reply:
and add a link to https://github.com/huggingface/blog/pull/2720 when it's merged aka. hf.co/blog/jfrog

Reply:
around "partnered with JFrog"